### PR TITLE
IGVF-1833-combine-audit-auxiliary

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/audit/measurement_set.py [hunk 1]
```diff
@@ -224,82 +224,8 @@ def audit_missing_institutional_certification(value, system):
...
-            detail = (
-                f'Measurement set {audit_link(path_to_text(value["@id"]),value["@id"])} '
-                f'has no cell sorting `auxiliary_sets`.'
-            )
-            yield AuditFailure('missing auxiliary set', f'{detail} {description}', level='NOT_COMPLIANT')
-
-
 @audit_checker('MeasurementSet', frame='object')
-def audit_missing_auxiliary_sets(value, system):
+def audit_missing_auxiliary_set_link(value, system):
```
Comment (ian-whaling) [inline src/igvfd/audit/measurement_set.py:228]: This audit concerns a different issue than the rest of the "missing auxiliary set" audits, so I did not include it in the unifying audit below.

**Comment by ian-whaling** [comment]
I just renamed and moved around the tests associated with these audits. Only additional test is the scQer related tests. Hope that helps review the code as it looks like a mess.

File: src/igvfd/audit/measurement_set.py [hunk 2]
```diff
@@ -471,3 +371,85 @@ def audit_missing_construct_library_set(value, system):
...
+            "audit_category": "missing auxiliary set",
+            "audit_level": "NOT_COMPLIANT"
+        },
+        {
+            "audit_description": "Variant FlowFISH measurement sets are expected to link to a variant sequencing auxiliary set.",
+            "audit_category": "missing auxiliary set",
+            "audit_level": "NOT_COMPLIANT"
+        },
+        {
+            "audit_description": "10X Multiome MULTI-seq assay measurement sets are expected to link to a oligo-conjugated lipids auxiliary set.",
```
Comment (jenjou) [inline src/igvfd/audit/measurement_set.py:406]: Minor typo, "an oligo" instead of "a oligo"

File: src/igvfd/tests/test_audit_measurement_set.py [hunk 3]
```diff
@@ -616,65 +616,99 @@ def test_audit_inconsistent_sequencing_kit(
...
     res = testapp.get(measurement_set['@id'] + '@@audit')
     assert any(
-        error['category'] == 'missing auxiliary set'
-        for error in res.json['audit'].get('NOT_COMPLIANT', [])
+        error['category'] == 'unexpected sample'
+        for error in res.json['audit'].get('ERROR', [])
+    )
+
+
+def test_audit_CRSIPR_screen_missing_files(
```
Comment (jenjou) [inline src/igvfd/tests/test_audit_measurement_set.py:648]: Minor typo "CRSIPR"

Comment (ian-whaling) [reply src/igvfd/tests/test_audit_measurement_set.py:648]: fixed

File: src/igvfd/audit/measurement_set.py (hunk 2 above)
Comment (ian-whaling) [reply src/igvfd/audit/measurement_set.py:406]: fixed.

**Review by jenjou — APPROVED** [review]

**Pull request merged**